src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -225,10 +223,8 @@ private StudentParticipation createNewParticipationWithInitializationDate(Exerci
         participation.setInitializationState(InitializationState.UNINITIALIZED);
         participation.setExercise(exercise);
         participation.setParticipant(participant);
-        // StartedDate is used to link a Participation to a test exam exercise
-        if (initializationDate != null) {
-            participation.setInitializationDate(initializationDate);
-        }
+        participation.setNumberOfAttempts(0);
```

Comment:
Is this necessary? numberOfAttemps should already be initialized with 0

Reply:
good point, I removed it